readthedocs/proxito/tests/test_full.py

```diff
@@ -471,6 +482,17 @@ def test_track_downloads(self, is_audit_enabled):
         self.assertEqual(log.action, AuditLog.DOWNLOAD)
 
 
+class ProxitoV2TestDocServingBackends(TestDocServingBackends):
+    def setUp(self):
+        super().setUp()
+        get(
+            Feature,
+            feature_id=Feature.USE_UNRESOLVER_WITH_PROXITO,
+            default_true=True,
+            future_default_true=True,
+        )
```

Comment:
Nice trick! 💯

Reply:
We should probably add a `# TODO` comment here to come back in the future and delete these classes.